mock-thunderhead/src/main/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementService.java

```diff
@@ -15,17 +15,14 @@
 import static com.sequenceiq.cloudbreak.auth.altus.model.Entitlement.CDP_AZURE_SINGLE_RESOURCE_GROUP_DEDICATED_STORAGE_ACCOUNT;
 import static com.sequenceiq.cloudbreak.auth.altus.model.Entitlement.CDP_AZURE_UAE_CENTRAL;
 import static com.sequenceiq.cloudbreak.auth.altus.model.Entitlement.CDP_BASE_IMAGE;
-import static com.sequenceiq.cloudbreak.auth.altus.model.Entitlement.CDP_CB_AWS_DISK_ENCRYPTION_WITH_CMK;
```

Comment:
I see that this entitlement is being checked in public API /api/v1/environments2//updateAwsDiskEncryptionParameters. if we remove it from mock thunderhead, we wont be able to test it locally until thunderhead is also updated to remove the same entitlement since mock thunderhead is used locally to set the entitlement for the account. can you check for this and other entitlements as well.

Reply:
The thunderhead PR is ready to get merged. @adarshsharma1983 , I hope your concern is addressed If we merge both thunderhead and CB PR's? Please let me know otherwise.

Reply:
if thunderhead PR will also be merged very soon then it will just be a matter of few days where feature won't work locally. other option is to revert mock thunderhead changes and do them only after thunderhead is merged. I don't know who works on these features and need to test locally.

Reply:
I have merged the thunderhead PR first and then only CB PR got merged,technically both CB and thunderhead PR's are merged now.

Reply:
if thunderhead changes are deployed first in production then feature wont work since entitlement will not be set and CB will check for entitlement.